Toolbar & headline section branch

ClintonYounge opened the pull request:
Features added-

1. The Toolbar and Headline Section; The index.html and style.css code with template styling.
2. Lint config for html and css.
3. A README.md template edited with the necessary information.
4. The background image and additional images.
5. A .gitignore file.

ClintonYounge committed:
Created .gitignore and added node_modules

ClintonYounge committed:
Updated .gitignore

ClintonYounge committed:
Added linter files

ClintonYounge committed:
Added README.md template

Microverse readme template

ClintonYounge committed:
Added the images needed

ClintonYounge committed:
Created index.html file

Created an index.html file and added html boilerplate

ClintonYounge committed:
Added font links to index.html

ClintonYounge committed:
Completed Toolbar Section

Added toolbar section. Necessary styles included.

ClintonYounge committed:
Added HTML for Headline Section

Added the necessary HTML to the index.html

ClintonYounge committed:
Added Headline Section CSS

Added the necessary CSS for the headline section.

Added target="blank" to the contact links.

Changed Pics Needed to Pics_Needed

ClintonYounge committed:
Updated my Readme

ClintonYounge committed:
Fixed typo in README

ClintonYounge committed:
Updated style.css to fix linter errors

ClintonYounge committed:
Updated style.css

Removed empty spaces to solve linter problems.

RaoAkif commented on README.md:
```diff
@@ -0,0 +1,137 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
[OPTIONAL] Your README is perfect 👍🏻 Just remove the Microverse Logo.

ClintonYounge committed:
Edited README.md

Fixed a typo.

RaoAkif commented on index.html:
```diff
@@ -0,0 +1,43 @@
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Toolbar and Headline Section</title>
```
[OPTIONAL] Change the title to what the Title of Project is. Just like for example your README. As README won't change with the feature you will be developing.
So You will be assigned custom names only to for example your feature branches, which would only stay relevant until you are done with that feature. But I loved the way you paid attention to even that title 👍🏻

RaoAkif commented on style.css:
```diff
@@ -0,0 +1,176 @@
+header nav div .menu-bar {
+  width: 18px;
+  height: 12px;
+  color: #6070ff;
```
`#6070ff` color is used 2 times & `#505f79` color is used 3 times in `style.css`. To make the code more DRY, you can create custom colors and use them again and again.
```
  --color-primary: #272a31;
  --color-secondary: #ec5242;
  --color-tertiary: #d3d3d3;
  --color-white: #fff;
  --color-grey: #585858;
  --color-light-grey: #ebf0ee;
```
or like
```
  --primary: #272a31;
  --secondary: #ec5242;
  --tertiary: #d3d3d3;
```

RaoAkif reviewed (changes requested):
## STATUS: REQUIRED CHANGES ♻️

Hi @ClintonYounge,
I really loved your work. But there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
_Check the comments under the review._
<img src="https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966" />

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Remove Micoverse Logo from README file
- Change the title of images folder from Pics_Needed to some like `images` or `images/icons` etc.
- Try not to repeat the same colors all over the style file. Try to write some DRY code. Create customize variables and use them.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ClintonYounge commented:
Thanks a lot for your timely response. I'll get to it!

On Fri, Jan 13, 2023, 12:28 PM Rao Muhammad Akif Tufail <
***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
> STATUS: REQUIRED CHANGES ♻️
>
> Hi @ClintonYounge <https://github.com/ClintonYounge>,
> I really loved your work. But to just push you a little bit, I will be
> suggesting some Required and Optional changes. You have done marvelous
> work, up till now. Happy Coding !!!
> Highlights
>
> ✔ Linters are on point
> ✔ Splendid work with the README file
> ✔ Great use of GitHub flow
> ✔ Good code design and implementation
> Required Changes ♻️
>
> *Check the comments under the review.*
>
> <https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966>
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> You can also consider:
>
>    - Remove Micoverse Logo from README file
>    - Change the title of images folder from Pics_Needed to some like
>    images or images/icons etc.
>    - Try not to repeat the same colors all over the style file. Try to
>    write some DRY code. Create customize variables and use them.
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In README.md
> <https://github.com/ClintonYounge/My-Portfolio-Project/pull/1#discussion_r1069636004>
> :
>
> > +<div align="center">
>
> +  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
>
>
> [OPTIONAL] Your README is perfect 👍🏻 Just remove the Microverse Logo.
> ------------------------------
>
> In index.html
> <https://github.com/ClintonYounge/My-Portfolio-Project/pull/1#discussion_r1069645673>
> :
>
> > @@ -0,0 +1,43 @@
>
> +<!DOCTYPE html>
>
> +<html lang="en">
>
> +  <head>
>
> +    <meta charset="UTF-8">
>
> +    <meta name="viewport" content="width=device-width, initial-scale=1.0">
>
> +    <meta http-equiv="X-UA-Compatible" content="ie=edge">
>
> +    <title>Toolbar and Headline Section</title>
>
>
> [OPTIONAL] Change the title to what the Title of Project is. Just like for
> example your README. As README won't change with the feature you will be
> developing.
> So You will be assigned custom names only to for example your feature
> branches, which would only stay relevant until you are done with that
> feature. But I loved the way you paid attention to even that title 👍🏻
> ------------------------------
>
> In style.css
> <https://github.com/ClintonYounge/My-Portfolio-Project/pull/1#discussion_r1069660973>
> :
>
> > +}
>
> +
>
> +header nav .logo {
>
> +  margin: 26px 223px 26px 0;
>
> +  font-weight: 700;
>
> +  font-size: 18px;
>
> +  line-height: 20px;
>
> +  align-self: center;
>
> +  color: #6070ff;
>
> +  white-space: nowrap;
>
> +}
>
> +
>
> +header nav div .menu-bar {
>
> +  width: 18px;
>
> +  height: 12px;
>
> +  color: #6070ff;
>
>
> #6070ff color is used 2 times & #505f79 color is used 3 times in style.css.
> To make the code more DRY, you can create custom colors and use them again
> and again.
>
>   --color-primary: #272a31;
>
>   --color-secondary: #ec5242;
>
>   --color-tertiary: #d3d3d3;
>
>   --color-white: #fff;
>
>   --color-grey: #585858;
>
>   --color-light-grey: #ebf0ee;
>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClintonYounge/My-Portfolio-Project/pull/1#pullrequestreview-1247924567>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4JSSCBKSQQZEQELJ5RHQV3WSF7B7ANCNFSM6AAAAAAT2RA63M>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

RaoAkif commented:
> 

👍🏻

ClintonYounge committed:
Reviewer instructed edits. Read desc.

1. Changed index.html title to "Clinton Younge's Portfolio"

2. Removed microverse logo from README

3. Edited colors in style.css

ClintonYounge committed:
Renamed Pics_Needed to images

ClintonYounge committed:
Customized Colors in style.css

ClintonYounge committed:
Updated css colors

ClintonYounge commented:
Hello. I made the instructed changes. Please let me know if there's any
more changes needed.

Thanks for your time.

On Fri, Jan 13, 2023, 12:36 PM Rao Muhammad Akif Tufail <
***@***.***> wrote:

> 👍🏻
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClintonYounge/My-Portfolio-Project/pull/1#issuecomment-1382101653>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4JSSCHEKB4G4U2O3TP4UHDWSF77FANCNFSM6AAAAAAT2RA63M>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

anagudelogu reviewed (approved):
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @ClintonYounge, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 

<img src="https://media0.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>

## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
